drivers/SmartThings/zwave-electric-meter/profiles/aeotec-home-energy-meter-gen8-1-phase-con.yml

```diff
@@ -0,0 +1,171 @@
+    preferenceType: enumeration
+    definition:
+      options:
+        0: "Disable"
+        1: "Enable"
+      default: 1
+  - name: imWThresholdTotal
+    title: "4. Import W threshold (total)"
+    description: "Threshold change in import wattage to induce an automatic report (Whole HEM)."
+    preferenceType: integer
+    definition:
+      minimum: 0
+      maximum: 60000
+      default: 50
+  - name: imWThresholdPhaseA
+    title: "5. Import W threshold (Phase A)"
+    description: "Threshold change in import wattage to induce an automatic report (Phase A)."
+    preferenceType: integer
+    definition:
+      minimum: 0
+      maximum: 60000
+      default: 50
+  - name: exWThresholdTotal
+    title: "8. Export W threshold (total)"
+    description: "Threshold change in export wattage to induce an automatic report (Whole HEM)."
+    preferenceType: integer
+    definition:
+      minimum: 0
+      maximum: 60000
+      default: 50
+  - name: exWThresholdPhaseA
+    title: "9. Export W threshold (Phase A)"
+    description: "Threshold change in export wattage to induce an automatic report (Phase A)."
+    preferenceType: integer
+    definition:
+      minimum: 0
+      maximum: 60000
+      default: 50
+  - name: imWPctThresholdTotal
+    title: "12. Import W threshold (total)"
+    description: "Percentage change in import wattage to induce an automatic report (Whole HEM)."
+    preferenceType: integer
+    definition:
+      minimum: 0
+      maximum: 100
+      default: 20
+  - name: imWPctThresholdPhaseA
+    title: "13. Import W threshold (Phase A)"
+    description: "Percentage change in import wattage to induce an automatic report (Phase A)."
+    preferenceType: integer
+    definition:
+      minimum: 0
+      maximum: 100
+      default: 20
+  - name: exWPctThresholdTotal
+    title: "16. Export W threshold (total)"
+    description: "Percentage change in export wattage to induce an automatic report (Whole HEM)."
+    preferenceType: integer
+    definition:
+      minimum: 0
+      maximum: 100
+      default: 20
+  - name: exWPctThresholdPhaseA
+    title: "17. Export W threshold (Phase A)"
+    description: "Percentage change in export wattage to induce an automatic report (Phase A)."
+    preferenceType: integer
+    definition:
+      minimum: 0
+      maximum: 100
+      default: 20
+  - name: autoRootDeviceReport
+    title: "32. Auto report of root device"
+    description: "Enable automatic report of root device."
+    preferenceType: enumeration
+    definition:
+      options:
+        0: "Disable"
+        1: "Enable"
+      default: 0
+  - name: defaultTime
+    title: "110. Set 111-116 to default"
+    description: "Set 111-116 to default."
+    preferenceType: enumeration
+    definition:
+      options:
+        0: "Disable"
+        1: "Enable"
+      default: 0
```

Comment:
This preference will most likely lead to problems, since we don't support reporting preference values from the device back up to the UI (two-way preferences). 

If the user resets the preference values to the default (i.e. sets this value to "Enable"), the UI will still display the old values, and not the defaults that are actually set on the device.